.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish to NPM
+
+on:
+  workflow_dispatch:
+    inputs:
+      deploy-sha:
+        description: 'SHA of the commit to deploy'
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      deploy-sha:
+        description: 'SHA of the commit to deploy'
+        required: true
+        type: string
+  
+jobs:
+  publish-to-npm:
+    name: Publish to NPM
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.deploy-sha }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.12.x
+          registry-url: https://registry.npmjs.org/
+          scope: '@constructorio'
+          token: ${{ secrets.NPM_TOKEN }}
+      - name: Install dependencies
+        run: npm ci
+      - name: Build compiled package for publishing
+        run: npm run compile
+      - name: Publish to NPM
+        run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Get new release version
+        id: get-version
+        run: echo "version=$(npm pkg get version)" >> "$GITHUB_OUTPUT"
+      - name: Notify Slack about new publish to NPM
+        uses: ravsamhq/notify-slack-action@v2
+        with:
+          notification_title: "New constructor-ui-autocomplete release published to npm at version ${{ steps.get-version.outputs.version }}"
```

Comment:
**Small suggestion:** Every other notification has emojis, should we add one here too?

```suggestion
          notification_title: "☁️ New constructor-ui-autocomplete release published to npm at version ${{ steps.get-version.outputs.version }}"
```

Reply:
Added in the latest commit!